chore(docs): add warn of ubuntu snap (#31)
koishijs/koishi-plugin-puppeteer: docs/index.md

为了正常使用这个插件，你首先需要确保你的电脑上已经安装有 Chrome / Edge / Firefox。同时，我们建议你保持浏览器和本插件的更新，因为版本不匹配可能会导致本插件无法正常使用。
:::

::: warning

> Ubuntu 在安装 Chrome 时可能会使用 snap 进行隔离。这将导致无法访问大部分主机文件夹或挂载点，并且 Koishi 可能会输出如下警告：

Reviewer: Considering change this text as:

> Since the software updater on Ubuntu install the Chromium and Chrome via [`snap`](https://snapcraft.io/), which isolates applications in their namespace sandboxes, which could cause several problems when we are using puppeteer to launch it.
> Koishi would throw an error below when you are using a `snap`-isolated Chromium-based browser:

> 由于 Ubuntu 中 `Software Updater` 会使用 [snap](https://snapcraft.io/) 安装 Chromium 与 Chrome。该方法会将应用隔离运行在 namespace 沙箱中。

Reviewer: > 该方法会将应用隔离运行在 namespace 沙箱中。

Weird Chinese.

Author: 想不出来 namespace 有什么更好的翻译（

> 当 Chromium 浏览器在隔离环境下使用时， Koishi 会抛出一下错误：

Reviewer 2: typo
```suggestion
当 Chromium 浏览器在隔离环境下使用时， Koishi 会抛出以下错误：
```

> 当 Chromium 浏览器在隔离环境下使用时， Koishi 会抛出一下错误：

Reviewer 3: ```suggestion
当 Chromium 浏览器在隔离环境下使用时， Koishi 会抛出以下错误：
```

[W] app Error: net::ERR_FILE_NOT_FOUND at file:///your-koishi-workspace-path/node_modules/koishi-plugin-puppeteer/index.html
```

可以进行如下方法解决：
- 使用传统 `deb` 包进行安装
    ```bash
    > curl -o https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb chrome.deb
    ...
    > dpkg chrome.deb

>     ...

Reviewer: What is these symbols? `...`?

Author: Ignored output, maybe I should delete it

    ```
- 将你想使用的文件夹添加到 `snap` 应用程序访问列表中
    ```bash
    sudo snap connect <snap app name>:your-koishi-workspace-path
    ```
- 将你想使用的文件移动到 `snap` 允许访问的文件夹或挂载点中（具体允许文件夹或挂载点请查询 snap 级别）

>   - `/home`: 用户挂载点

Reviewer: ```suggestion
  - `/home/$USER`: 用户家目录
```

为了正常使用这个插件，你首先需要确保你的电脑上已经安装有 Chrome / Edge / Firefox。同时，我们建议你保持浏览器和本插件的更新，因为版本不匹配可能会导致本插件无法正常使用。
:::

::: warning

> 由于 Ubuntu 中 `Software Updater` 会使用 [snap](https://snapcraft.io/) 安装 Chromium 与 Chrome。这会将应用隔离运行在一个 namespace 沙箱中。

Reviewer 2: ```suggestion
由于 Ubuntu apt 存储库中没有 Chromium 包，只能通过 [snap](https://snapcraft.io/) 安装 Chromium 或 Chrome。这会将 Chromium 隔离运行在一个沙箱中，只能访问受限的文件系统。
```

> 由于 Ubuntu apt 存储库中没有 Chromium 包，只能通过 [snap](https://snapcraft.io/) 安装 Chromium 或 Chrome。这会将 Chromium 隔离运行在一个沙箱中，只能访问受限的文件系统。

Reviewer: ```suggestion
由于 Ubuntu 在 19.10 以后转向[通过 snap 提供 Chromium 包](https://ubuntu.com/blog/chromium-in-ubuntu-deb-to-snap-transition)，snap 会将 Chromium 隔离运行在一个沙箱中，只能访问受限的文件系统。
```

当 Chromium 浏览器在隔离环境下使用时， Koishi 会抛出以下错误：
```bash
[W] app Error: net::ERR_FILE_NOT_FOUND at file:///your-koishi-workspace-path/node_modules/koishi-plugin-puppeteer/index.html
```

<details>
  <summary>点我查看解决方法</summary>

可以选择下列方法**之一**解决：
- 使用其他 Linux 发行版

> - 不使用 `Software Updater`

Reviewer: This redundant line can be removed.

> - 不使用 `Software Updater`

Reviewer: Add one more method:

> 使用 [`nix`](https://nix.dev/install-nix) 安装
>     ```bash
>     nix-env -iA chromium
>     ```

Reviewer: Ignore this
